src/include/duckdb/common/vector_operations/vector_operations.hpp

```diff
@@ -87,17 +87,19 @@ struct VectorOperations {
 	// Select Comparisons
 	//===--------------------------------------------------------------------===//
 	static idx_t Equals(Vector &left, Vector &right, const SelectionVector *sel, idx_t count, SelectionVector *true_sel,
-	                    SelectionVector *false_sel);
+	                    SelectionVector *false_sel, ValidityMask *null_mask = nullptr);
```

Comment:
Could these pointers to selection vectors be `optional_ptr`s instead?

Reply:
They could - I was just matching the existing style. Do we change all pointers, just the new ones, or leave it for another cleanup pass?

Reply:
I'll start with the new argument then. My concern about the existing arguments was just a concern that I would have to change all the callers.